Source/Initialization/WarpXAMReXInit.cpp

```diff
@@ -24,6 +24,15 @@ namespace {
         bool abort_on_out_of_gpu_memory = true; // AMReX' default: false
         pp.query("abort_on_out_of_gpu_memory", abort_on_out_of_gpu_memory);
         pp.add("abort_on_out_of_gpu_memory", abort_on_out_of_gpu_memory);
+
+        // If warpx.numprocs is used for the domain decomposition, we set the blocking factor to 1
+        // so that the asserts that the number of cells is a multiple of blocking factor always pass
```

Comment:
Wouldn't setting `blocking_factor` to `1` also potentially influence performance by a lot?

Reply:
I think that if we use `warpx.numprocs` the blocking factor is not used to define the grids so its value should have no effect during the simulation. @WeiqunZhang let me know if I am wrong .

Reply:
Great, I was hoping for that, if that is the case let us add this side-effect in the comment here and also in `Docs/source/running_cpp/parameters.rst`, please.

Reply:
Yes this is much clearer this way, thank you!